lib/season_statistics.rb

```diff
@@ -69,9 +103,23 @@ def fewest_tackles
         end
            
             team_fewest_tackles = tackles_teams.min_by { |team, tackles| tackles }[0]
-    end
-
+    
+            find_team_name(team_fewest_tackles)
+        end
+        def find_team_name(team_id)
+            #get the corresponding team name to the team id number in teams...
+        end
     def most_tackles
+        tackles_teams = Hash.new(0)
+        @game_data.each do |game|
+            team_id = game[:team_id]
+
+            tackles_teams[team_id] += game[:tackles].to_i
+        end
+           
+            team_most_tackles = tackles_teams.max_by { |team, tackles| tackles }[0]
+    
+            find_team_name(team_most_tackles)
```

Comment:
We have a method for team name in the updated stat_tracker you can pull from

Reply:
This is why it is important to make small updates (aka 1 method) and make sure you do frequent pull requests to your local machine from main.  It keeps you up do date with the code everyone else on the time is writing.